lib/galaxy/tool_util/deps/mulled/util.py

```diff
@@ -64,13 +64,19 @@ def _namespace_has_repo_name(namespace, repo_name, resolution_cache):
     if resolution_cache is not None and cache_key in resolution_cache:
         repo_names = resolution_cache.get(cache_key)
```

Comment:
Shouldn't the cache have different `repo_names` for different `namespace`s?

Reply:
I think so, this would be a problem if we we're using more than one namespace, but so far it's only biocontainers.

Reply:
Yep, added to https://github.com/galaxyproject/galaxy/issues/13460 for tracking.